test/exnumerable_test.exs

```diff
@@ -14,20 +14,18 @@ defmodule ExnumeratorTest do
   test "should return a random value",
     do: assert Message.sample in @values
 
-  test "should augment given types" do
-    assert Message.cast("sent")      == {:ok, :sent}
-    assert Message.cast(:read)       == {:ok, :read}
-    assert Message.load("received")  == {:ok, :received}
+  test "should argument given types" do
+    assert Message.cast("sent")      == {:ok, "sent"}
+    assert Message.load("received")  == {:ok, "received"}
     assert Message.dump("delivered") == {:ok, "delivered"}
-    assert Message.dump(:sent)       == {:ok, "sent"}
+    assert Message.cast(:sent)       == :error
+    assert Message.load(:received)   == :error
+    assert Message.dump(:delivered)  == :error
```

Comment:
These errors should not be handled in this test, shouldn't they? We are checking for positive results here. The other test checks for errors.

Reply:
This is actually the most important part, since this branch will break backward compatibility. Users no longer pass atom lists.
But you're right. This test specs won't be asserted. Test specs in their individual apps should assert the changes.

Reply:
I don't want to remove this test but to separate successful checks from the errored ones in a separate example.